src/main/java/ViewModel/playlistViewModel.java

```diff
@@ -1,4 +1,37 @@
 package ViewModel;
+import Controller.newPlaylistController;
+import Entities.RegularUser;
+import Entities.Song;
+import InputData.newPlaylistInputData;
+import OutputData.newPlaylistOutputData;
+import UseCase.createPlaylist;
 
+
+import Presenter.playlistPresenter;
 public class playlistViewModel {
+    private final newPlaylistController newPlaylistController;
+    private  playlistPresenter playlistPresenter;
+
+    public playlistViewModel(int id, RegularUser user, String playlistName, Song firstSong){
+        // get necessary information from View to construct a new playlist
+        newPlaylistInputData inputData = new newPlaylistInputData(id,playlistName,firstSong,user);
+        //Blank output data object to extract success message from
+        newPlaylistOutputData outputData = new newPlaylistOutputData("");
+        this.playlistPresenter = new playlistPresenter(this, outputData);
+//        newPlaylistInputBoundary newPlaylistInputBoundary = InputBoundary.newPlaylistInputBoundary;
+        createPlaylist newPlay= new createPlaylist();
+        this.newPlaylistController = new newPlaylistController(newPlay,inputData);
+        //TODO: Does this violate clean architecture ?
+    }
+
+    /**
+     *
+     * @return output message generated by the Controller and update the presenter's outputMessage with it as well
+     */
+    public String callNewPlaylistUseCase(){
+        String msg =  this.newPlaylistController.createPlaylist();
+        newPlaylistOutputData outputData = new newPlaylistOutputData(msg);
```

Comment:
I'm a bit confused as to what you're trying to do in this function. Could you add comments? 

Also, see my comment on playlistPresenter -- if you pass the string message in from the Presenter as a parameter, you can just return it immediately.